includes/class-wc-structured-data.php

```diff
@@ -275,38 +274,46 @@ public function generate_product_data( $product = null ) {
 				'reviewCount' => $product->get_review_count(),
 			);
 
-			// Markup most recent rating/review.
+			// Markup 5 most recent rating/review.
 			$comments = get_comments(
 				array(
-					'number'      => 1,
+					'number'      => 5,
 					'post_id'     => $product->get_id(),
 					'status'      => 'approve',
 					'post_status' => 'publish',
 					'post_type'   => 'product',
 					'parent'      => 0,
-					'meta_key'    => 'rating',
+					'meta_key'    => 'rating', // phpcs:ignore WordPress.DB.SlowDBQuery.slow_db_query_meta_key
 					'orderby'     => 'meta_value_num',
+					'meta_query'  => array( // phpcs:ignore WordPress.DB.SlowDBQuery.slow_db_query_meta_query
```

Comment:
I wonder if we need this comparison. Rating will always be numeric right? Are you using this as a way to check if rating actually exists?

Reply:
Yes, checking if is not just some comment without any rating.